docs/OV_Runtime_UG/deployment/deployment_intro.md

```diff
@@ -24,17 +24,33 @@ Once the [OpenVINO application development](../integrate_with_your_application.m
 
 The table below shows which distribution type can be used depending on target operation system:
 
+@sphinxdirective
+
+.. raw:: html
+
+    <div class="collapsible-section" data-title="Click to expand/collapse">
+
+@endsphinxdirective
+
 | Distribution type | Operation systems |
 |------- ---------- | ----------------- |
 | Debian packages | Ubuntu 18.04 long-term support (LTS), 64-bit; Ubuntu 20.04 long-term support (LTS), 64-bit |
 | RMP packages | Red Hat Enterprise Linux 8, 64-bit |
-| Docker images | All operation systems |
+| Docker images | Ubuntu 18.04 long-term support (LTS), 64-bit; Ubuntu 20.04 long-term support (LTS), 64-bit; Red Hat Enterprise Linux 8, 64-bit; Windows Server Core base LTSC 2019, 64-bit; Windows 10, version 20H2, 64-bit |
 | PyPi (pip package manager) | All operation systems |
```

Comment:
@slyubimt what OSes we need to provide for pip? could you please clarify the exact list?
Here we don't have the exact list of supported OSes https://docs.openvino.ai/nightly/openvino_docs_install_guides_installing_openvino_pip.html, suppose we need to update it as well.

Reply:
In general OpenVINO Runtime can be installed on:

* Windows 10*
* macOS* 10.15 
* Ubuntu* 18.04,  Ubuntu* 20.04 and RHEL 8
manylinux_2_27 tag allows to install on any Linux OS with GLIBC >=2.27, but we do not provide validation for these systems, except mentioned.

It would be better to provide a link to a single place where it is always updated: https://pypi.org/project/openvino/